alembic/versions/62b9fc80f1fe_add_soft_delete.py

```diff
@@ -0,0 +1,50 @@
+"""Add soft delete
+
+Revision ID: 62b9fc80f1fe
+Revises: c53edf85cda4
+Create Date: 2019-01-25 15:51:33.926243
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '62b9fc80f1fe'
+down_revision = 'c53edf85cda4'
```

Comment:
This down revision references a the wrong file. There is a new file on develop with the number `c8a866f85fd7` that it should show instead.
@RachaelNantale has tried it out on her p.c as well and got the same error